### DISCUSSION TITLE
Using an iframe to render a pdf

### DISCUSSION THREAD

**Discussion by willking98** [surveydown-dev/surveydown#99] [Feature Requests]
### Description

Is it possible to render an iframe showing a pdf (for example) on a survey page? Below is the code I have used successfully in other Quarto documents.

```{=html}
<iframe width="780" height="500" src="https://css4.pub/2015/icelandic/dictionary.pdf" title="Information"></iframe>
```

**Comment by jhelvy** [comment]
I just tried that exact example and I'm seeing it inside the app. Just drop the iframe in.

<img width="857" alt="Screenshot 2024-09-24 at 5 04 59 PM" src="https://github.com/user-attachments/assets/263951b7-8a37-49e2-999a-3f8ac7f49e8c">

**Reply by willking98** [reply]
Sorry for that! Not rendering in the RStudio window but working fine when opened in the browser!
